lib/matplotlib/fontconfig_pattern.py

```diff
@@ -177,19 +177,34 @@ def _property(self, s, loc, tokens):
 parse_fontconfig_pattern = lru_cache()(FontconfigPatternParser().parse)
 
 
+def _escape_val(val, escape_func):
```

Comment:
Would be nice to have docstring describing what this does an what the parameters / returned value are.